server/app/repository/VersionRepository.java

```diff
@@ -277,15 +277,25 @@ public boolean isDraft(Question question) {
         .anyMatch(draftQuestion -> draftQuestion.id.equals(question.id));
   }
 
+  /** Returns true if the program is a member of the current draft versions. */
   public boolean isDraft(Program program) {
     return isDraftProgram(program.id);
   }
 
+  /** Returns true if the program with the provided id is a member of the current draft versions. */
   public boolean isDraftProgram(Long programId) {
     return getDraftVersion().getPrograms().stream()
         .anyMatch(draftProgram -> draftProgram.id.equals(programId));
   }
 
+  /**
+   * Returns true if the program with the provided id is a member of the current active versions.
```

Comment:
**praise:** 
```suggestion
   * Returns true if the program with the provided id is a member of the current active version.
```

Reply:
Singular because `Version` is a specific resource in CiviForm and there is only one active Version at a time